src/libraries/Microsoft.XmlSerializer.Generator/src/Sgen.cs

```diff
@@ -431,7 +461,7 @@ private void ImportType(Type type, ArrayList mappings, ArrayList importedTypes,
             }
             if (xmlTypeMapping != null)
             {
-                xmlTypeMapping = importer.ImportTypeMapping(type);
+                xmlTypeMapping = importer.ImportTypeMapping(type, defaultNamespace);
```

Comment:
Also redundant.

Reply:
See above.
Due to the way XmlReflectionImporter is designed, providing the 'defaultNamespace' argument to ImportTypeMapping is not redundant.